frappe/commands.py

```diff
@@ -986,7 +986,8 @@ def set_config(context, key, value):
 @click.argument('site')
 @click.option('--root-login', default='root')
 @click.option('--root-password')
-def drop_site(site, root_login='root', root_password=None):
+@click.option('--archive-path')
```

Comment:
Let's use `--archived-sites-path` and `archived_sites_path`. Explicit and the variable that we are going to use.

At line 1005, you just need to write `if not archived_sites_path: archived_sites_path = ...`